cartridges/openshift-origin-cartridge-jbossews/bin/tomcat

```diff
@@ -87,42 +83,84 @@ EOF
 done
 
 # Control the Tomcat instance
-if [ "$1" = "start" ]; then
-  ${JAVACMD} $JAVA_OPTS $LOGGING_CONFIG $CATALINA_OPTS \
-    -classpath "$CLASSPATH" \
-    -Dcatalina.base="$CATALINA_BASE" \
-    -Dcatalina.home="$CATALINA_HOME" \
-    -Djava.endorsed.dirs="$JAVA_ENDORSED_DIRS" \
-    -Djava.io.tmpdir="$CATALINA_TMPDIR" \
-    org.apache.catalina.startup.Bootstrap start 2>&1
-    if [ ! -z "$CATALINA_PID" ]; then
-      echo $! > $CATALINA_PID
-    fi
-elif [ "$1" = "start-security" ]; then
-  ${JAVACMD} $JAVA_OPTS $LOGGING_CONFIG $CATALINA_OPTS \
-    -classpath "$CLASSPATH" \
-    -Dcatalina.base="$CATALINA_BASE" \
-    -Dcatalina.home="$CATALINA_HOME" \
-    -Djava.endorsed.dirs="$JAVA_ENDORSED_DIRS" \
-    -Djava.io.tmpdir="$CATALINA_TMPDIR" \
-    -Djava.security.manager \
-    -Djava.security.policy=="${CATALINA_BASE}/conf/catalina.policy" \
-    org.apache.catalina.startup.Bootstrap start 2>&1 
-    if [ ! -z "$CATALINA_PID" ]; then
-      echo $! > $CATALINA_PID
-    fi
-elif [ "$1" = "stop" ]; then
-  ${JAVACMD} $JAVA_OPTS $LOGGING_CONFIG \
-    -classpath "$CLASSPATH" \
-    -Dcatalina.base="$CATALINA_BASE" \
-    -Dcatalina.home="$CATALINA_HOME" \
-    -Djava.endorsed.dirs="$JAVA_ENDORSED_DIRS" \
-    -Djava.io.tmpdir="$CATALINA_TMPDIR" \
-    org.apache.catalina.startup.Bootstrap stop 2>&1
-elif [ "$1" = "version" ]; then
-  ${JAVACMD} -classpath ${CATALINA_HOME}/lib/catalina.jar \
-    org.apache.catalina.util.ServerInfo
+if [ -n "$JAVA_OPTS_EXT" ]; then
+  if [ "$1" = "start" ]; then
+      ${JAVACMD} $JAVA_OPTS "$JAVA_OPTS_EXT" $LOGGING_CONFIG $CATALINA_OPTS \
```

Comment:
Will quoting `$JAVA_OPTS_EXT` fix the general problem? The Bugzilla report uses the following command in the steps to reproduce:
```console
$ rhc env set JAVA_OPTS_EXT="-Dworking.dir=app-root/data/test dir" -a APP_NAME
```
After this change, using this first example, you will indeed pass `-Dworking.dir=app-root/data/test dir` as a single argument to `${JAVACMD}`.  However, consider a second example:
```console
$ rhc env set JAVA_OPTS_EXT="-Dfoo=bar -Dbaz=quux" -a APP_NAME
```
Now we will pass `-Dfoo=bar -Dbaz=quux` as a single argument to `${JAVACMD}`.  Is this second example something we should support? Don't we need to split `$JAVA_OPTS_EXT` in the second example?